rust/hyperlane-base/src/types/multisig.rs

```diff
@@ -10,6 +11,18 @@ use hyperlane_core::{
 };
 
 use crate::{CheckpointSyncer, CoreMetrics};
+/// Weights are scaled by 1e10 as 100%
+pub type Weight = u64;
```

Comment:
nit: imo type aliases like this can be more confusing than just using u64 but happy either way

Reply:
It's nice to be explicit that we want the Weight here instead we want u128/u64 which doesn't tell why?